elevate-cpanel

```diff
@@ -3338,9 +3338,25 @@ sub _system_update_check() {
 
     INFO("Checking if your system is up to date: ");
     ssystem(qw{/usr/bin/yum clean all});
-    if ( ssystem(qw{/usr/bin/yum check-update}) != 0 ) {
+
+    my $out = ssystem_capture_output(qw{/usr/bin/yum check-update -q});
+
+    if ( $out->{status} != 0 ) {
+
+        # not a blocker: only a warning
         WARN("Your system is not up to date please run: /usr/bin/yum update");
```

Comment:
I don't understand why this isn't a blocker. doesn't leapp freak out if you're not up to date on 7 before you go to 8?